src/elm/clinical.ts

```diff
@@ -202,12 +215,12 @@ export class ConceptDef extends Expression {
 export class ConceptRef extends Expression {
   name: string;
 
-  constructor(json: any) {
+  constructor(json: ELM.ConceptRef) {
     super(json);
-    this.name = json.name;
+    this.name = json.name || '';
```

Comment:
ugh you know the drill at this point. My fault again, property can be optional. It's almost like I should've read the dang [manual](https://cql.hl7.org/04-logicalspecification.html#conceptref) when doing this conversion. Sorry

Reply:
I personally don't hate defaulting to empty strings because usually you just do a check like ```
if (json.name) {
   // Do Something
}
```
and null or empty string behave the same way. It also makes the code and types simpler sometimes without having to always check for null in cases where you want to do something like 
```
json.name.toLowerCase()
```
But the broader discussion goes back to whether there are cases where we'd actually want to update the XML spec itself and change certain properties to required or not. There are pros and cons to having the single source of truth for the ELM spec. But it could get messy if we were to say manually start editing the generated ELM Typescript Types, because then we would not be able to generate it again later when the XML schema changes.

Reply:
At this point, unless there is a strong reason for it, I'd shy away from introducing changes to the ELM specification. Those changes are expensive given the governance process on CQL.  In cases where something isn't clearly wrong, it may be better just to accommodate for it.